src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -1555,21 +1556,21 @@ public bool IsExecutingInvocation(string invocationId)
             return _executingInvocations.ContainsKey(invocationId);
         }
 
-        public bool TryFailExecutions(Exception workerException)
+        public void Shutdown(Exception workerException)
         {
-            if (workerException == null)
-            {
-                return false;
-            }
+            WorkerShutdownException shutdownException = workerException is not null
```

Comment:
Want to make sure I understand the core behavior change of this PR:

_Before_: some cases would call this with a `null` workerException, and thus this was a no-op.
_After_: those same places may still call with `null` workerException, but it is no longer a no-op and will shut the channel down.

Am I understanding this right?

Reply:
Before `TryFailExecutions` itself was a no-op and this never happens

```
invocation.Context?.ResultSource?.TrySetException(workerException);
RemoveExecutingInvocation(invocationId);
```

But channel is still gone/disposed regardless